src/main/app/court-finder/court.ts

```diff
@@ -3,7 +3,7 @@ import { Address } from './address'
 export class Court {
   constructor (
     public readonly name: string,
-    public readonly distance: number,
+    public readonly slug: string,
```

Comment:
slug?

Reply:
yes slug :), don't know why its called that but its something like this `    slug: 'birmingham-district-probate-registry',
`

Reply:
It's the "Slugified" version of the court name